docs/pages/guides/typescript.mdx

```diff
@@ -0,0 +1,83 @@
+import PageMeta from '_shared/PageMeta';
+
+<PageMeta
+  title="Typescript – @material-ui/pickers"
+  description="A guide of how to use @material-ui/pickers with typescript"
+/>
+
+## TypeScript guide
+
+Pickers are originally written in typescript, so type definetions are provided with the code without any additional requirements.
```

Comment:
It don't see the link between written in TypeScript and not required additional requirement. What about:

```suggestion
The date picker components come with built-in TypeScript definitions.
```